android/app/build.gradle

```diff
@@ -133,10 +133,10 @@ android {
     }
 
     defaultConfig {
-        applicationId 'com.zingo'
+        applicationId 'com.ZingoMobile'
```

Comment:
I think this would need a very broad change?

Reply:
We had a name collision in the app store with a pre-existing `zingo`.

Reply:
I can change locally every time I upload a new version of the App... no problem. I can rollback this right away...

Reply:
You may as well keep in the name change.  Unless @skyl has a better idea for the `applicationId`?

Reply:
I just assume that all of the `com.zingo` instances would need to be change

<img width="707" alt="Screen Shot 2022-07-21 at 1 29 59 PM" src="https://user-images.githubusercontent.com/61438/180309478-e3e45262-7f7a-4fc4-89f9-f6b9ecb43504.png">
d

Reply:
also, `com.foo` should be lower case according to convention, not sure if it an actual rule tho.

Reply:
https://developer.android.com/studio/build/configure-app-module#set_the_application_id
The AplicationID is nothing to do with name of the package, could be the same or not. In my case, I create the App before you @skyl change everything to `zingo` and I don't want to delete the app and create other in my google play space with `com.zingo`. Since this is only for testing... sorry, but I don't change it. I'm going to close this PR right now, and I did all of this locally every time there are a new version...